nssdb/metadata.go

```diff
@@ -0,0 +1,48 @@
+package nssdb
+
+import (
+	"context"
+	"fmt"
+)
+
+type Metadata struct {
+	ID    string
+	Item1 []byte
+	Item2 []byte
+}
+
+type Password struct {
+	Salt              []byte
+	EncryptedPassword []byte
+}
+
+// The schema of the metaData table is (id string, item1, item2)
+func (db *NSSDB) GetMetadata(ctx context.Context, id string) (*Metadata, error) {
+	entry := &Metadata{
+		ID: id,
+	}
+	err := db.Key.QueryRowContext(ctx, "SELECT item1, item2 FROM metaData WHERE id = $1", id).Scan(&entry.Item1, &entry.Item2)
```

Comment:
`$1` or `?`? The examples in the repo use `?`,

Reply:
2168975